service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/ReturnedCaseContentProvider.java

```diff
@@ -36,7 +37,9 @@ private ReturnedCaseTemplate.ReturnedCaseTemplateBuilder templateData(CaseData c
         ReturnApplication returnApplication = caseData.getReturnApplication();
 
         return ReturnedCaseTemplate.builder()
-            .localAuthority(laLookup.getLocalAuthorityName(caseData.getCaseLocalAuthority()))
+            .localAuthority(nonNull(caseData.getCaseLaOrRelatingLa())
```

Comment:
oh I think this should always be the getApplicantName() regardless of getCaseLaOrRelatingLa - the getLocalAuthorityName might return the respondent LA name, rather than the 3rd party applicant